pkg/graphql/response.go

```diff
@@ -0,0 +1,8 @@
+package graphql
+
+import "encoding/json"
+
+type responseBody struct {
+	Data   *json.RawMessage        `json:"data"`
```

Comment:
With the `json.RawMessage` type, we can leave the `data` object unmarshaled, so we can unmarshal it later in the context that calls the client.

For example:
https://github.com/giantswarm/kubectl-gs/blob/e78a4abf48c2284760121938d6a79dbc60a850de/pkg/installation/info.go#L49

Reply:
Maybe for inspiration: in my implementation I'm unmarshalling directly to the object provided https://github.com/giantswarm/config-controller/blob/948f76c29f9da500b4f7ab307203cdafbced4458/pkg/github/internal/graphql/client.go#L36

Reply:
That's actually quite interesting, and closer to how built-in functions behave. I'll refactor this to match that 👍🏻 

Also, I had no idea you already built a GraphQL client... I would've just copied that 😅